### ISSUE TITLE
Sublime Merge prevent Windows 11 task bar from auto showing

### ISSUE THREAD

**Issue by safield** [sublimehq/sublime_merge#1849]
**Version info**

Windows 11
Sublime Merge 2091

When Windows 11 task bar is set to auto hide/show, and Sublime Merge is maximized, the task bar will not auto show when the mouse cursor is put to the bottom of the screen.

When testing with other application in maximized mode, the task bar will show when the mouse is moved to the bottom of the screen.

**Comment by safield** [comment]
HELLO....Hello....hello.....

Lonely in here.

**Comment by might42** [comment]
Yep, I had reported this previously here: https://github.com/sublimehq/sublime_merge/issues/1794
But its not like it is any more busy over there.

**Comment by might42** [comment]
> When testing with other application in maximized mode, the task bar will show when the mouse is moved to the bottom of the screen.

SublimeText does not show this behavior here, either. Yet, there are other application behaving equally bad, e.g. the Loom screen recording app I need to use in my job (not that I would want to compare the considerably high overall quality of Sublime Merge with that rubbish piece of Loom software).
